cdci_data_analysis/analysis/email_helper.py

```diff
@@ -268,6 +270,30 @@ def send_job_email(
                                              'attachment',
                                              filename="api_code.py")
 
+    status_details_message = None
+    status_details_title = status
+    if status_details is not None and status_details['status'] != 'successful':
+        if status_details['status'] == 'empty_product':
+            status_details_message = '''Unfortunately, after a quick automated assessment of the request, it has been found that it contains an <b>empty, product</b>.
+To the best of our knowledge, no unexpected errors occurred during processing,
+and if this is not what you expected, you probably need to modify the request parameters. We are sorry.<br>'''
+            status_details_title = 'finished: with empty product'
+        elif status_details['status'] == 'empty_result':
+            status_details_message = '''Unfortunately, after a quick automated assessment of the request, it has been found that it contains an <b>empty, result</b>.
+To the best of our knowledge, no unexpected errors occurred during processing,
+and if this is not what you expected, you probably need to modify the request parameters. We are sorry.<br>'''
+            status_details_title = 'finished: with empty result'
+        elif status_details['status'] in ['exception', 'connection_error', 'dispatcher_exception']:
+            status_details_message = '''Unfortunately, after a quick automated assessment of the request, a problem has been found.
```

Comment:
I do not this this should be ever sent to user. The message self-contradicting: problem, but no errors. 
If the error is indeed unexpected, we should get a sentry and that's it - expect our reaction. 

The only thing is that I am not sure if some reasonable cases, some untreated empty outcomes, fall into this category. Maybe let's keep it as sentry only and see.

Reply:
ok I agree, nothing to the user for the moment, but will add a `TODO` for potential future adjustments

Reply:
`TODO` added